docs/source/mods/metromap.rst

```diff
@@ -0,0 +1,511 @@
+for each edge, a direction (0 to 7) is computed, and all nodes are assigned to x-
+and y-coordinates that support the edge directions. Additionally, the following
+is considered
+
+* The original positions of the nodes are respected in the following way:
+
+  * Only directions that align with the original direction or are
+    directly neighbored are allowed. For example, if v is north-east (1) from u in the original
+    positions, it is allowed to place v north, north-east, or east in the
+    octilinear representation, i.e., for the edge :math:`(u,v)` the directions
+    2, 1, and 0 are allowed.
+  * The ordering of all adjacent nodes of a node :math:`v` is preserved.
+* The distance (difference in x- or y-coordinate) between each two nodes is at
+  least one and as minimal as possible
+* If a set of paths in the graph is given (representing, for example, metro lines),
+  the bends along the lines are minimized.
+* If possible, ensure planarity.
+
+Some remarks about planarity: The number of constraints to ensure planarity is
+quadratic in the number of edges of the graph. Hence, the planarity part is the
+largest part of the model. Instead of adding all the constraints initially, we
+use a callback approach and lazily add these constraints, i.e., only when they
+are violated. Moreover, the planarity restrictions are included as soft
+constraints, i.e., if there is no planar representation, an embedding with the
+fewest planarity violations is computed. There is an option to turn off the
+planarity requirement. The restrictions from the origin geographical positions
+often imply a planar solution without including planarity constraints. In these
+cases, the performance can be improved if planarity constraints are not
+considered.
+
+We consider three weights to balance the different parts of the objective. The
+weight :math:`w_o` is a penalty for a direction of an edge :math:`(u,v)`. More
+precisely, it penalizes the direction if :math:`v` is not placed in the original
+but at a neighboring direction. The weight :math:`w_d` penalizes a distance of
+an edge greater than a minimum distance of one, and :math:`w_b` penalizes the
+line bends. These three weights have a default value of 1 but are parameterized
+and can be adapted. Additionally, the planarity violations are included in the
+objective. They are weighted with 1000. This should be considered when chosing
+weights for :math:`w_o`, :math:`w_d`, and :math:`w_b`; we restrict the values to
+be between 0 and 100.
+
+
+Background: Mathematical Formulation
+------------------------------------
+
+This OptiMod is implemented by formulating a Linear Program (LP) and solving it
+using Gurobi. Instead of presenting the complete formulation as a whole (which
+is too bulky), the different aspects of the model are discussed below.
+
+
+.. dropdown:: Variables
+
+  We use the following variables:
+
+    - :math:`d(u,v,r)\in\{0,1\}` for :math:`(u,v)\in E` and :math:`r\in\{0,1,\ldots,7\}`
```

Comment:
Should this be aligned with the line above? It renders as a quote but maybe this is intended. When aligned, looks like this: 
<img width="863" alt="Screenshot 2024-06-14 at 16 17 00" src="https://github.com/Gurobi/gurobi-optimods/assets/23246013/128fcabe-8b16-4d86-aa89-ede94bf81a57">

Reply:
Done